GeorgiyYegorashchenko/HW4/task1.py

```diff
@@ -0,0 +1,5 @@
+input_value = int(input('Enter the temperature in Celsius: '))
```

Comment:
Variable Naming: The variable name input_value is okay, but it might be more descriptive to name it something like temperature_in_celsius. This makes the code more readable.